tests/TileDB.CSharp.Test/QueryTest.cs

```diff
@@ -400,5 +400,85 @@ public void TestNullableAttributeArrayQuery()
 
         }// public void TestNullableAttributeArrayQuery
 
+        [TestMethod]
+        public void TestBoolAttributeArrayQuery()
+        {
+            // Create array
+            var context = Context.GetDefault();
+            Assert.IsNotNull(context);
+
+            var dim1 = Dimension.Create<int>(context, "rows", new [] { 1, 2 }, 2);
+            var dim2 = Dimension.Create<int>(context, "cols", new [] { 1, 2 }, 2);
+            var domain = new Domain(context);
+            domain.AddDimensions(dim1, dim2);
+
+            var a1 = new Attribute(context, "a1", DataType.TILEDB_BOOL);
+
+            var array_schema = new ArraySchema(context, ArrayType.TILEDB_DENSE);
+            array_schema.AddAttribute(a1);
+            array_schema.SetDomain(domain);
+            array_schema.Check();
+
+            var tmpArrayPath = Path.Join( Path.GetTempPath(), "tiledb_test_bool_array");
+            if (Directory.Exists(tmpArrayPath))
```

Comment:
Side note - please make a story to create a fixture for this. In the future we will want to be able to run against non-local URIs and ensure, so centralizing temp directory logic will simplify things a lot.